### DISCUSSION TITLE
Few Features Requested.

### DISCUSSION THREAD

**Discussion by udaymehta** [iSoron/uhabits#712] [Feature Requests]
Hi @iSoron, 

I want to suggest/if-possible few features.

-  A **Simple Note** system with in-built To-do List feature or like a journal. You can take a look at this [noteapp](https://github.com/gsantner/markor), which is opensource with inbuilt markdown support.
 
    _Why this feature?_
      - Notes are important as we can keep track of everything not only related to habits, but it can be anything. Like how was this X day or some study related note etc.
  

-  A **Pomodoro timer** which is important part of forming habits.
 
   _Why this feature?_
    - It can be useful for making timer for study, programing, playing video games etc. it can be used in many ways.
    

_(OPTIONAL)_

-  A **Donation Button** in Settings (Which directly links to your respected banks or Paypal Account)

   _Why this feature?_
     - I have seen many users who likes to donate developers including me for there work.

Hope at least one of this feature is added in future build.

**Comment by hiqua** [comment]
Daily notes could be interesting.

I feel that the pomodoro timer does not belong here, and there are already many apps doing it.

**Comment by drawtheliner** [comment]
+1 for a simple notes system.

As long as the app remains lightweight and fast!
